rpc-rs/src/channel_log.rs

```diff
@@ -7,13 +7,16 @@
 //!
 #![cfg(not(target_arch = "wasm32"))]
 
-use log::{Level, Metadata, Record};
-use once_cell::sync::OnceCell;
 use std::{
     fmt,
+    io::Write,
+    str::FromStr,
     sync::atomic::{AtomicUsize, Ordering},
 };
 
```

Comment:
sigh. we're using different rustfmt styles. I always keep these together so that they are sorted (even std). No need to change them back.

Reply:
Actually this isn't a rustfmt style difference as much as it is a style difference. I was always taught to group imports by std, external crates, and internal imports (in that order, though that matters less). Definitely makes the imports easier to find IMO and to know what is internal vs external